.travis.yml

```diff
@@ -205,3 +218,9 @@ script:
       bjam arm-tests warnings-as-errors=on ssl=$ssl crypto=$crypto variant=test_arm $toolset target-os=linux link=static testing.launcher="sudo cp -R bin rootfs/; sudo chroot rootfs";
     fi'
   - cd ..
+
+# test build for iOS, using x86_64 to avoid the need to build boost from source for arm
+  - 'if [ "$iphone" == "1" ]; then
+      brew upgrade boost;
+      bjam -j3 variant=test_debug warnings-as-errors=on asserts=on toolset=darwin-x86_64~iphonesim link=static;
```

Comment:
this is just building. shouldn't you ``cd`` into tests, to run the tests as well?
Perhaps the binaries you get from iphonesim aren't executable directly from a macOS shell, is that the case?